src/net/server/handlers/mul.rs

```diff
@@ -92,33 +93,22 @@ pub async fn handler(
         step,
     };
 
-    let body = hyper::body::to_bytes(req.body_mut()).await?;
-    let envelopes = body
-        .as_ref()
-        .chunks(headers.data_size as usize)
```

Comment:
so we should remove `data_size` no? will still need `offset` to handle out-of-order messages until we get streaming solution